app/serializers/user_action_serializer.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class UserActionSerializer
+  include JSONAPI::Serializer
+
+  attribute :user_action_event, :status, :subject_user_verification, :acting_ip_address,
+            :acting_user_agent, :created_at, :updated_at, :acting_user_verification
```

Comment:
the relationships should have id at the end. 
```suggestion
  attribute :user_action_event_id, :status, :subject_user_verification_id, :acting_ip_address,
            :acting_user_agent, :created_at, :updated_at, :acting_user_verification_id
```